charts/apisix/values.yaml

```diff
@@ -312,6 +312,21 @@ logs:
   errorLog: "/dev/stderr"
   errorLogLevel: "warn"
 
+# access and error log configuration
+nginx:
+  core:
+    worker_rlimit_nofile: 20480
+    events:
+      worker_connections: 10620
+  logs:
+    enableAccessLog: true
```

Comment:
How does those `logs` configuration item work in the current version? Maybe need some compatibility code?

Reply:
I think we need to change all the places that we use the log parameters.